guides/deep-dives/field-editors.md

```diff
@@ -1,7 +1,119 @@
+![An editor panel on the right edge, with field inputs](/images/card-sdk/right-edge-example.jpg)
+
+It is a WYSIWYG (what you see is what you get) experience where a user makes changes to the Card's content and sees immediately what the result would look like.
+
+## Enabling the Right Edge
+
+To start using the right edge, first make sure that you have the following in your project's `application.hbs`:
+
+```handlebars
+{{#cardstack-edges}}
+  {{#squishable-container}}
+    {{outlet}}
+  {{/squishable-container}}
+{{/cardstack-edges}}
+```
+
+For someone to see the editing panel, the must have a valid user session and the right grants that allow them to edit content. You can learn more about authorization, authentication, and mock users for testing in [Grants](../../data/grants/).
+
+## Built-in field editors
+
+Field editors are components that handle user input.
+Cardstack provides some built-in content editor components.
+
+When a field is set to certain `core-types` in a Card's schema, some editors get automatically applied:
+
+- Boolean: `field-editors/boolean-editor` is automatically provided for `@cardstack/core-types::boolean`
+- Date: `field-editors/date-editor` is automatically provided for `@cardstack/core-types::date`
+- Integer: `field-editors/integer-editor` is automatically provided for `@cardstack/core-types::integer`
+- String: `field-editors/string-editor` is automatically provided for `@cardstack/core-types::string`
+
+Other built-in editors should be set for each field in the Card's schema: 
+
+- Datetime: use `field-editors/datetime-editor`
+- Dropdown: use 
+    - `field-editors/dropdown-choices-editor`
+    - `field-editors/dropdown-multi-select-editor`
+    - `field-editors/dropdown-search-editor`
+    - `field-editors/dropdown-search-multi-select-editor`
+
+For example, this dropdown menu lets someone assign a `category` grouping to a photo Card:
+
+```js
+    factory.addResource('fields', 'category').withAttributes({
+      fieldType: '@cardstack/core-types::belongs-to',
+      editorOptions: { displayFieldName: 'name' },
+      editorComponent: 'field-editors/dropdown-choices-editor'
+})
+```
+
+## Passing options to a field editor
+
+Field editors can take options using the `editorOptions` property in a Card's schema for a specific field:
+
+```js
+factory.addResource('fields', 'is-admin-user').withAttributes({
+  fieldType: '@cardstack/core-types::boolean',
+  editorOptions: { style: 'switch' }
+})
+```
+
```

Comment:
Suggestion: Add a sentence saying something like, "In this example, a property named `style` with the value `switch` will get passed to the `field-editors/boolean-editor` component."

I think this kind of redundancy will help readers more quickly grok the concepts

Reply:
Thank you, added!